### PR TITLE
Add User class #235 #236

### PR DESCRIPTION
**Pull request by valentinbujdoso**

### CODE REVIEW

File: src/main/java/org/rulez/demokracia/pdengine/dataobjects/UserEntity.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import org.rulez.demokracia.pdengine.persistence.BaseEntity;
+
+public abstract class UserEntity extends BaseEntity {
+
+	private static final long serialVersionUID = 1L;
+	public String proxyId;
+	@ElementCollection
+	public List<String> assurances;
+
+	public UserEntity() {
```
Comment (magwas) [inline src/main/java/org/rulez/demokracia/pdengine/dataobjects/UserEntity.java]: Do we actually need a default constructor here?

File: src/test/java/org/rulez/demokracia/pdengine/UserTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	@tested_feature("Supporting functionality")
+	@tested_operation("User")
+	@tested_behaviour("the user has a proxy id")
+	@Test
+	public void check_proxy_id() {
+		User user = new User("Test");
+		assertEquals("Test", user.proxyId);		
+	}
+	
+	@tested_feature("Supporting functionality")
```
Comment (magwas) [inline src/test/java/org/rulez/demokracia/pdengine/UserTest.java:32]: This tests for the empty set, which is okay.
Please test for nonempty set as well.

**Review by magwas — CHANGES_REQUESTED** [review]

File: src/test/java/org/rulez/demokracia/pdengine/UserTest.java (hunk 2 above)
Comment (valentinbujdoso) [reply src/test/java/org/rulez/demokracia/pdengine/UserTest.java:32]: Do you mean this?

**Review by magwas — APPROVED** [review]
good work

**Comment by magwas** [comment]
testcaseCount=3

**Pull request merged**